gym/lib/gym/runner.rb

```diff
@@ -29,7 +29,11 @@ def run
         path
       elsif Gym.project.mac?
         compress_and_move_dsym
-        copy_mac_app
+        if Gym.project.command_line_tool? || Gym.project.command_line_library?
```

Comment:
As mentioned in the other PR, are we sure that this check for `command_line_library` is not going to include false positives when building frameworks?